molecule/default/molecule.yml

```diff
@@ -13,3 +13,7 @@ platforms:
     pre_build_image: true
 provisioner:
   name: ansible
```

Comment:
What are the advantages of having it on molecule in this case? I added a separate lint workflow because with molecule the same checks were repeated in each distro for molecule, resulting in longer wait in Github Actions. Am I missing something?

Reply:
@staticdev the benefit would be that one would always run the linting everyting one run molecule during development on the local computer and could fix lint issues as soon as they occur whithout needing the extra step of running lint explicitly.

Reply:
@MorganLindqvist I understand your point but for me does not justify the time on pipeline. With precommit I forget about lint and focus on my change while developing, then I worry when I do the commits. Also for me is bad to change one role to lint different since I have a standard already for many roles. Hope you can also see my side since I have extra costs involved to maintain the role.

Reply:
@staticdev sure, will remove lint from the molecule build